blink/procfs.c

```diff
@@ -20,7 +20,10 @@
 
 #include <fcntl.h>
 #include <stdlib.h>
+
+#if !defined(__STDC_NO_THREADS__) || __STDC_NO_THREADS__ == 0
 #include <threads.h>
+#endif
```

Comment:
```suggestion

```

`_Thread_local` is a standard C11 keyword and does not require including this header. It is only required when using the convenience macro `thread_local`.